modules/source/file/config/file.xml

```diff
@@ -39,7 +39,20 @@
 			</header-enricher>
 			<file:file-to-bytes-transformer />
 		</chain>
+	</beans:beans>
 
+	<beans:beans profile="use-contents-with-split">
+		<chain id="extractContentsAndSplit" input-channel="files" output-channel="output">
+			<header-enricher>
+				<header name="contentType" value="text/plain"/>
```

Comment:
I guess not but it anyway goes to `bufferedReader.readLine()` method in SI so do we really have any other option?